source/colors.js

```diff
@@ -15,106 +15,108 @@ module.exports = {
     //      * value: the color as a six-digit hex value; eg, '#acefed'
 
     {
-      name: 'cool blue',
+      name: 'ultramarine',
+      synonyms: 'cool blue',
       values: [
-        { tone: 'core', value: '#2a51b8' },
-        { tone: '0', value: '#e5e9ff' },
-        { tone: '10', value: '#cdd7ff' },
-        { tone: '20', value: '#abbdff' },
-        { tone: '30', value: '#84a2ff' },
+        { tone: 'core', value: '#3151b7' },
+        { tone: '1', value: '#e7e9f7' },
+        { tone: '10', value: '#d1d7f4' },
+        { tone: '20', value: '#b0bef3' },
+        { tone: '30', value: '#89a2f6' },
         { tone: '40', value: '#648fff' },
         { tone: '50', value: '#3c6df0' },
-        { tone: '60', value: '#2a51b8' },
-        { tone: '70', value: '#254190' },
-        { tone: '80', value: '#203065' },
-        { tone: '90', value: '#1b2449' }
+        { tone: '60', value: '#3151b7' },
+        { tone: '70', value: '#2e3f8f' },
+        { tone: '80', value: '#252e6a' },
+        { tone: '90', value: '#20214f' }
       ]
     },
     {
       name: 'blue',
       values: [
-        { tone: 'core', value: '#1d79ca' },
-        { tone: '0', value: '#e2eafd' },
-        { tone: '10', value: '#c7d9fb' },
-        { tone: '20', value: '#a0c1f9' },
-        { tone: '30', value: '#63a8ff' },
-        { tone: '40', value: '#3c97f4' },
-        { tone: '50', value: '#1d79ca' },
-        { tone: '60', value: '#2c5a90' },
-        { tone: '70', value: '#224874' },
-        { tone: '80', value: '#1d3452' },
+        { tone: 'core', value: '#2d74da' },
+        { tone: '1', value: '#e1ebf7' },
+        { tone: '10', value: '#c8daf4' },
+        { tone: '20', value: '#a8c0f3' },
+        { tone: '30', value: '#79a6f6' },
+        { tone: '40', value: '#5392ff' },
+        { tone: '50', value: '#2d74da' },
+        { tone: '60', value: '#1f57a4' },
+        { tone: '70', value: '#25467a' },
+        { tone: '80', value: '#1d3458' },
         { tone: '90', value: '#19273c' }
       ]
     },
     {
-      name: 'warm blue',
+      name: 'cerulean',
+      synonyms: 'warm blue',
       values: [
         { tone: 'core', value: '#009de2' },
```

Comment:
This looks like a bigger issue across multiple colors of which tone is considered core. I will look into it.